doc/man1/ciphers.pod

```diff
@@ -723,6 +722,27 @@ The following names are accepted by older releases:
 Some compiled versions of OpenSSL may not include all the ciphers
 listed here because some ciphers were excluded at compile time.
 
+=head1 BUGS
+
+The B<DEFAULT> string does not actually follow described syntax as
+descibed in CIPHER LIST FORMAT section.  In particular, instead of
+behaving as I<cipher string>, it works as prefix of the cipherlist
+which enables default ciphers.  This means that it may only
+appear at start of the cipherlist, it may not be combined with other
+cipherstrings using the B<+> character and any characters following
+the prefix will be treated as separate cipherstring (regardless of
+delimiters).
+
+For example, according to rules, B<DEFAULT+ECDH> should be interpreted as
+intersection of the default list and ciphers that contain ECDH algorithm.
+However, it will be actually interpreted as cipherstring B<DEFAULT>,
+followed by B<+ECDH>, i. e. it will enable default ciphers AND all ECDH
+ciphers (just as B<DEFAULT:+ECDH> would).
```

Comment:
+ECDH does not enable ECDH ciphers, it just moves them to the end. Just "ECDH" would add them, but the !COMPLEMENTOFDEFAULT doesn't allow adding any ciphers not in DEFAULT.

Reply:
Nice catch.  I'll fix it in a sec. (See below for comment about the `!`)